### PR TITLE
Rnd 759 client parameters to store

### PR DESCRIPTION
**Pull request by liranbahar**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/common/superstream/Superstream.java [hunk 1]
```diff
@@ -69,6 +51,7 @@ public class Superstream {
     public String ConsumerSchemaID = "0";
     public Map<String, Descriptors.Descriptor> SchemaIDMap = new HashMap<>();
     public Map<String, Object> configs;
+    private Map<String,?> allClientConfigs;
```
Comment (shay23b) [inline clients/src/main/java/org/apache/kafka/common/superstream/Superstream.java:54]: call it fullClientConfigs + modify methods to be with same name

**Review by shay23b — CHANGES_REQUESTED** [review]
* Add the missing imports to project(':clients') in build.gradle
* Update version in superstream folder to 3.5.113

File: clients/src/main/java/org/apache/kafka/common/superstream/Superstream.java (hunk 1 above)
Comment (liranbahar) [reply clients/src/main/java/org/apache/kafka/common/superstream/Superstream.java]: done

**Review by shay23b — CHANGES_REQUESTED** [review]
send configs of admin client as well

**Review by shay23b — APPROVED** [review]

**Pull request merged**